codalab/model/bundle_model.py

```diff
@@ -1267,6 +1268,8 @@ def make_condition(field, value):
         for row in rows:
             row = str_key_dict(row)
             row['group_permissions'] = uuid_group_permissions[row['uuid']]
+            if row['title']:
+                row['title'] = self.decode_str(row['title'])
```

Comment:
This takes care of worksheet fetching through 'wsearch' command line, the None check is needed as some legacy title might be None for some reason (i.e before didn't prevent title being None)